### ISSUE TITLE
other minor lab improvements

### ISSUE THREAD

**Issue by e-dorigatti** [slds-lmu/lecture_i2dl#11]
- [x] (lab1 ex1): mention that `(-5, -10, 10)` is but one of the correct parametrizations, there are more that are fine (and show some)
 - [ ] (lab 2): the url for the blum book doesn't lead anywhere.
 - [ ] (lab 4): sometimes people don't get that gradients measure input sensitivity and that this the very idea of backprop. perhaps a small calculation for, say, a regression problem with one essentially irrelevant feature, could help get the idea across
 - [ ] (lab 5): for the weight decay exercise, maybe you could borrow a plot from the fcim slides on that, might help people not so familiar with regu
 - [ ] (lab 7): remove weird image overlap on the last page (pdf)
 - [ ] (lab 9): add more on modern rnns, perhaps something on computing lstm cell states could be helpful
 - [ ] (lab 11): add elbo derivation as an exercise
